### DISCUSSION TITLE
checkoutComplete doesn't allow to buy products with partial translations. null value in column \"translated_product_name\" violates not-null constraint\nDETAIL

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#10728] [General] [answered]
Hi,
If I try to checkout products that have a translated description,
I'm blocked to the payment step and customer cannot complete the order.

After that the only way to continue to sell the product is remove it from Saleor and reinsert it.

Enviroment: 
dashboard v3.1.0-dev
core v3.8.0-a

This is the response of the gql after the checkoutComplete mutation:

```
{
  "errors": [
    {
      "message": "null value in column \"translated_product_name\" violates not-null constraint\nDETAIL:  Failing row contains (Eyeguard EG-S 1010 Blu, ewfwegewrg, 1, 204.800, 204.800, t, 0, 43, 0.0000, null, EUR, , L / Verde, 204.800, 204.800, 51.200, 51.200, Sale: U2FsZToy, fixed, 256.000, 256.000, 256.000, 256.000, f, UHJvZHVjdFZhcmlhbnQ6NDM=, U2FsZToy, null, e8619966-7250-41ef-b90e-ad4d4423f8e6, 1c6ed79e-6e32-434e-812e-36d69b778cb1, null, 2022-09-26 07:21:48.189276+00, 204.800, 256.000, {}, {}).\n",
      "locations": [
        {
          "line": 152,
          "column": 3
        }
      ],
      "path": [
        "checkoutComplete"
      ],
      "extensions": {
        "exception": {
          "code": "IntegrityError"
        }
      }
    }
  ],
  "data": {
    "checkoutComplete": null
  },
  "extensions": {
    "cost": {
      "requestedQueryCost": 14,
      "maximumAvailable": 50000
    }
  }
}
```

Reproducing this issue is simple:  create a product and add a translation only for the description (Italian language).

Can you have a check please ?

 Thanks and kind regards,
 
 Davide

**Comment by JustCreateDeveloper** [comment]
Exactly the same problem here. For the moment I'll have to create a translation for each product name (otherwise customers cannot place an any order)

**Comment by timuric** [comment, marked as answer]
Could you please try latest stable version instead?
